src/Codegen/Builders/InputTypeBuilder.hack

```diff
@@ -1,5 +1,5 @@
 namespace Slack\GraphQL\Codegen;
```

Comment:
Some of the changes here are because we had never run `hackfmt` on the whole codebase before.